DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/userpreferences/userpreferences.controller.js

```diff
@@ -9,15 +9,26 @@
 
     function UserPreferencesController(userData, logger, API_URL) {
         var vm = this;
+        vm.imageSource = ''; 
 
         activate();
 
         function activate() {
+            uploadPhoto();
             logger.info('Activate User Preferences View');
         }
 
+        function uploadPhoto() {
+            return userData.getImage(API_URL.UPLOADPHOTO).then(function (newImageUrl) {
```

Comment:
2 questions.
- URL is called `API_URL.UPLOADPHOTO` and function is `uploadPhoto` but actually we are performing GET request. Why?
- controller should not be aware of the API routes. so API_URL should be included to service (not controller)